go.mod

```diff
@@ -12,6 +13,8 @@ require (
 	github.com/mitchellh/cli v1.1.5
 	github.com/rogpeppe/go-internal v1.11.0
 	github.com/russross/blackfriday v1.6.0
+	github.com/yuin/goldmark v1.6.0
```

Comment:
I'm a proponent of using this Markdown library, but I do think we should strive to only use one Markdown library in the Go module. Can you create an issue to switch over and remove blackfriday? I think there's also a blackfriday/v2 issue that we could close too.